### DISCUSSION TITLE
Problems when exporting packages to JSON

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [microsoft/winget-cli#1264] [Q&A]
When I try to export my apps to JSON so I can import them later, I get a lot of these errors:
```
Installed package is not available from any source: Microsoft Edge Dev
```

What's going on? This happens for a lot of packages, even ones I installed through WinGet.

**Comment by denelon** [comment]
@TheOnlyTails I created #1274 to track this one. There might be a bug, or something else going on.

**Reply by TheOnlyTails** [reply]
Great, thank you!
